src/ml/neural_net/mps_layer_helper.mm

```diff
@@ -0,0 +1,90 @@
+#include <ml/neural_net/mps_layer_helper.h>
+#include <ml/neural_net/mps_weight.h>
+#include <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation TCMPSLayerHelper
+
++ (MPSCNNFullyConnectedNode *) createFullyConnected:(MPSNNImageNode *)inputNode
+                               inputFeatureChannels:(NSUInteger)inputFeatureChannels
+                              outputFeatureChannels:(NSUInteger)outputFeatureChannels
+                                        inputHeight:(NSUInteger)inputHeight
+                                         inputWidth:(NSUInteger)inputWidth
+                                            weights:(float *)weights
+                                             biases:(float *)biases
+                                              label:(NSString *)label
+                                      updateWeights:(bool)updateWeights
+                                             device:(id<MTLDevice>)dev
+                                          cmd_queue:(id<MTLCommandQueue>) cmd_q {
+  
+  turi::neural_net::OptimizerOptions optimizerOptions;
+
+  TCMPSConvolutionWeights *FullyConnectedDataLoad = [[TCMPSConvolutionWeights alloc] initWithKernelWidth:inputWidth
```

Comment:
Do me a favor and just start the entire RHS of this assignment on the next line (indented 4 over), e.g.
```
TCMPSConvolutionWeights *FullyConnectedDataLoad =
    [[TCMPSConvolutionWeights alloc] initWithKernelWidth:inputWidth
...
```

Reply:
alright, done!